shared/src/commonTest/kotlin/com/tajemniktv/tajsos/data/RepositoryDelegationTest.kt

```
@@ -73,8 +75,10 @@ class RepositoryDelegationTest {
         override suspend fun updateNode(node: NodeEntity) {
         }
 
+
         override suspend fun deleteNode(node: NodeEntity) {
         }
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)